example/glue/configure_includes/configure_linux_riscv.mk

```diff
@@ -21,17 +21,19 @@
 
 include $(CONFIG_INCL_DIR)/configure_common.mk
 
-RISCV_CROSSTOOLS_PREFIX=riscv64-unknown-linux-gnu
-
 ifneq (,$(findstring _riscv64, $(SPEC)))
-	RISCV_TARGET=riscv64-unknown-linux-gnu
+  RISCV_CROSSTOOLS_PREFIX ?= riscv64-unknown-linux-gnu
 else
-	RISCV_TARGET=riscv32-unknown-linux-gnu
+  ifeq (install, $(RISCV_TOOLCHAIN_TYPE))

```

Comment:
Where would `RISCV_TOOLCHAIN_TYPE` originate?
Why not just provide a default for ``RISCV_CROSSTOOLS_PREFIX as is done for the 64-bit case?

Reply:
Updated as suggested above. So use can refer this to set up the variable in OMR.